### ISSUE TITLE
Opened image files are garbled

### ISSUE THREAD

**Issue by abandon-hope** [photopea/photopea#6677]
Your Operating System: Windows 11
Your Web Browser: Firefox, Chrome, Edge (tested across all)
The problem happens in the Incognito Mode, too: Yes

**This must be done to see the problem**
1. Go to www.photopea.com
2. Open an image file (PNG,JPG, etc)
3. If image is not garbled, repeat - it's intermittent but fairly reliable
4. Garbled images are either black if opened as a new image, or a random mix of transparent and magenta, typically in bands, if imported into an existing image via eg: drag-and-drop from the filesystem.

I can appreciate this can't be happening to everyone or you'd get more bug requests on it, but that's all I have to do, and it's been going on for a few weeks. If I clear the local cache/cookies for photopea.com, the bug will abate for one import, before starting again.

**Comment by photopea** [comment]
What is "garbled"? I just opened eight JPG images in Photopea and they all look correctly.

**Comment by Noriych** [comment]
> What is "garbled"? I just opened eight JPG images in Photopea and they all look correctly.

![zx](https://github.com/photopea/photopea/assets/168057535/446d5920-b973-473b-b9a9-9b00517caef4)

Maybe this is what he meant, this also happened to me

**Comment by camoody1** [comment]
@Noriych He's definitely not going to be able to help you until you disable or remove whatever extension you have installed that is blocking ads. It takes some guts to ask for support from a guy who you're blocking the ads of.

**Comment by Noriych** [comment]
> @Noriych He's definitely not going to be able to help you until you disable or remove whatever extension you have installed that is blocking ads. It takes some guts to ask for support from a guy who you're blocking the ads of.

![zzz](https://github.com/photopea/photopea/assets/168057535/52844041-08fb-4376-a870-3f5bf7c11564)

Oh sorry that was my mistake, I've turned it off. I tried different (High res 4000x3000px PNG) images but the problem seems to be the same

**Comment by camoody1** [comment]
@Noriych Can you post one of the images you're having trouble with here? Or is it just all images? It would probably help to include information about your PC hardware, too. CPU, GPU (if using one), amount of RAM. Also, under your "More" menu, does anything change if you enable/disable the "Use WebGL" option?

**Comment by Noriych** [comment]
> @Noriych Can you post one of the images you're having trouble with here? Or is it just all images? It would probably help to include information about your PC hardware, too. CPU, GPU (if using one), amount of RAM. Also, under your "More" menu, does anything change if you enable/disable the "Use WebGL" option?

I think i found the problem, it only happens when i import PNG images with 64 bit depth

Yes, there is "Use WebGL" option, i turn it on/off but nothing change.

And im using i5 10400f, rtx 3060, 16gb ram

![xxx](https://github.com/photopea/photopea/assets/168057535/caf0cf80-78c9-467d-9599-0b9e9d8b55f5)

**Comment by camoody1** [comment]
@Noriych Ivan is probably going to want to see an example of these 64-bit PNGs so he can work it on his end. Maybe send the image to him via support@photopea.com with this Issue # in the subject referencing this issue.

**Comment by photopea** [comment]
Could you share your image (a file) with us? I could solve it a long time ago if you provided your file.

**Comment by Noriych** [comment]
> Could you share your image (a file) with us? I could solve it a long time ago if you provided your file.

Here is the sample file
![Tin Can (64 Bit Depth)](https://github.com/photopea/photopea/assets/168057535/a71da2c4-9013-49c2-908b-4532478c1420)

**Comment by Noriych** [comment]
> Could you share your image (a file) with us? I could solve it a long time ago if you provided your file.

Here is the sample file
![Tin Can (64 Bit Depth)](https://github.com/photopea/photopea/assets/168057535/4610f8ba-91fc-4fbf-91fc-d5c0ccdc8261)

**Comment by photopea** [comment]
It happened only if you inserted specific 16-bit PNG images into an existing 16-bit document. It should be fixed now.

The original report confused be, because it definitely never happened for JPG images. And it never happened after opening an image (only after inserting an image into another document).

**Comment by photopea** [comment]
As I said, it should be fixed now. Have you tried it today?

**Comment by abandon-hope** [comment]
Oh yeah, I took those screenshots as I posted that.

**Comment by abandon-hope** [comment]
I also just checked and it still happens if WebGL is disabled.

**Comment by photopea** [comment]
It does not happen to me when the WebGL is disabled :( could you tell me a short list of steps what should I do to see it "garbled"?

**Comment by abandon-hope** [comment]
Well, the website is now coming up as a blank grey screen (see screenshot below) but the instructions I gave in my initial submission are the same steps I followed to produce the earlier screnshot. I can open PSD files, but if I tried to import a PNG file it would appear as above - bands of magenta and transparent - whether WebGL was on or off, and whether in Incognito mode or not. It doesn't happen 100% of the time, but once it's happened once it'll happen repeatedly with any file imported from thereon out. Sometimes it's fixed itself when I come back to the site, sometimes not - so I've tried doing things like clearing cached data and cookies for this URL, which has sometimes fixed it, but sometimes not.

![image](https://github.com/photopea/photopea/assets/73901973/96b3138c-7a51-4de7-8c4c-114399c3b5e2)

**Comment by photopea** [comment]
Are you sure you are opening www.photopea.com and not something else? Did you try it in the Incognito mode? Nobody is reporting any problems with Photopea, and there was over 1 million users in the last 24 hours.

**Comment by abandon-hope** [comment]
Yeah - I was hesitant to ever post a bug because I figured if it was an actual bug, surely you'd know about it. The grey screen thing doesn't happen in Edge browser (I can't check Chrome right now) but is persistant in Firefox.  As you can see from the banner in the screenshot, I am visiting www.photopea.com. 

Currently imports are working in Edge, for the record - but it's fairly typical for the import bug to only show up after a few imports - rarely, it won't occur at all.

**Comment by abandon-hope** [comment]
I'm going to bed but I realised you asked for the other guy's specs so I'll give mine in case there's anything useful:

Intel(R) Core(TM) i5-9400F @ 2.90GHz
8.00 GB RAM
Windows 11 Home / 64-bit operating system, x64-based processor
NVidia GTX-1660 GPU 6GB GDDR5

**Comment by photopea** [comment]
I can not see www.photopea.com in any of your screenshots :( 

I can open www.Photopea.com in Firefox just fine. Did you try to update your firefox? How many browser extensions do you use in Firefox? Why do you cut away the browser UI in every screenshot, so that we can not tell what URL are you at? :D and what browser / extensions you are using.

**Comment by abandon-hope** [comment]
You can see it's Photopea in the first screenshot. I see photopea.com in the text of the last screenshot in three places. My habit is to use region grabber since it's the default. For what it's worth, here it is again, showing the full window, where you can see it is photopea.com.

![image](https://github.com/photopea/photopea/assets/73901973/4a2f6a55-5cd8-4ef4-ac6c-7f5f46e71fee)

All taken in Firefox in Private Browsing mode, with no extensions active. In Edge I have no extensions.

**Comment by abandon-hope** [comment]
Also no updates pending on FireFox. Figured I'd include the URL bar, just to stay one step ahead of the game.
![image](https://github.com/photopea/photopea/assets/73901973/e1b363d6-254a-4736-8b85-3fe40466d4f8)

**Comment by photopea** [comment]
Here is what I see at Photopea.com in the Incongito mode of Firefox.

![image](https://github.com/photopea/photopea/assets/15955558/56afaefa-7b91-4a9f-afb0-3418048464c5)

Do you see any errors in a web console when openint Photopea? In what country are you located? Have you tried a different internet provider? Maybe there is some firewall on your network.

**Comment by photopea** [comment]
What if you turn on a wifi hotspot on your phone and connect to the internet through your phone?

**Comment by abandon-hope** [comment]
There's no signal here, just the nationally-provided fiber-optic line.

**Comment by photopea** [comment]
Photopea is used by 60,000 users from the UK every day. Nobody reports any problem. I think it is caused by your internet provider.

**Comment by abandon-hope** [comment]
The vast majority of that 60K have their internet served by the same provider though?

**Comment by photopea** [comment]
I dont know. I guess they have different providers.

**Comment by abandon-hope** [comment]
British Telecom have the largest single market share (33%), and are responsible for all broadband infrastructure in the UK.

**Comment by photopea** [comment]
Could you try to open Photopea on a different device, e.g. on a phone?

**Comment by abandon-hope** [comment]
Seems to be fine - hard to verify the import thing as it's an intermittent issue, but I imported a dozen different images and they all worked, and no grey screen ofcourse. The grey screen is just FireFox.

**Comment by photopea** [comment]
You see a grey screen at www.Photopea.com in firefox in your phone?

**Comment by abandon-hope** [comment]
No - just FIreFox on desktop. I only have Chrome on my phone, so that's all I tested.

**Comment by photopea** [comment]
Do you have any other devices where you could test www.photopea.com ?

**Comment by abandon-hope** [comment]
Unfortunately not - just my desktop and phone. I'd feel confident saying it's something to do with my desktop, but I don't know what. I haven't installed any new software in a long while, or made any hardware changes. I thought it might be to do with a GPU driver update but I've had another since the bug started and no change. Is there anything else local to a single device that could affect how the site works?

Oh - I also realised I made a folder with my works-in-progress when I first hit the bug, so I have an exact record of when it started: ‎07 ‎April ‎2024, ‏‎15:59:15

**Comment by abandon-hope** [comment]
Honestly - I wouldn't blame you if you wanted to just throw this one out, since it seems to be limited to one single device. Can't win them all 😅 I'm sure I can find other ways to make my workflow work.

**Comment by photopea** [comment]
Did you try www.Photopea.com in Google Chrome on your desktop?

**Comment by abandon-hope** [comment]
Yeah - no grey screen, same import bug.

**Comment by photopea** [comment]
So what is this import bug? Nobody else is reporting it. Does it happen when you open www.photopea.com in Chrome Incognito?

**Comment by abandon-hope** [comment]
The import bug is the one shown in the first screenshot I provided, also reproduced below. It's a fresh install of Chrome, and it's not signed in - would it make any difference if it was Incognito? I only ask because it can sometimes take a while to reproduce the bug.

![325724463-9130aba4-611d-4d03-a549-ca576b4ab38f](https://github.com/photopea/photopea/assets/73901973/20830140-d6c0-4fb5-820f-e3aed91e9b52)

**Comment by photopea** [comment]
Yes, try it in Incognito

**Comment by abandon-hope** [comment]
I can't replicate it in Incognito so far, having tried just going through my usual workflow, and periodically importing a bunch of images to see if I can produce a garbled one, for the last 20 minutes or so. I'll come back to this test when I've got a day to put aside to doing texture-work, cos it pretty reliably shows up within a few hours.

**Comment by photopea** [comment]
I think you have been using the old version of Photopea from three days ago. Did you keep www.photopea.com opened in your browser for three days without closing it? You need to close Photopea and open it again, to use the latest version.

As I said, I fixed it three days ago, so if you closed and opened it again three days ago, it should have been fixed. As you did in Incognito right now.

**Comment by abandon-hope** [comment]
My computer's been restarted multiple times since then (we don't have 100% reliable power here) - once because I read you'd issued a fix. I cleared caches for photopea.com before that restart, just to be sure. After restart, in a fresh Incognito window as per the advice, I opened my document, tried to import something, captured the screenshot, and came back here to report the problem was ongoing. Since then I've also re-installed Chrome, and replicated the bug there.

**Comment by photopea** [comment]
As I undrestand, everything works well in Chrome Incognito now, is that rigth? Could you try it in the regular mode of Chrome now, too?

**Comment by abandon-hope** [comment]
No I've replicated the bug in Chrome in the last 2 days, as said in the previous message.

**Comment by photopea** [comment]
Okay. But if a website works differently in Incognito than in a regular mode, it is usually caused by your browser extensions.

**Comment by abandon-hope** [comment]
It is a new installation of Chrome, and I've already reproduced it outside of Incognito mode.

**Comment by photopea** [comment]
Ok, let me know if there is anything I can help you with.

**Comment by abandon-hope** [comment]
With the import bug across all browsers, or with the grey screen on Firefox?

**Comment by photopea** [comment]
I got confused with your requests. If you still have an issue with Photopea, could you start a new issue and describe the steps to reproduce it?

**Comment by abandon-hope** [comment]
You want me to copy and paste this thread into a new one?

**Comment by photopea** [comment]
Also, report only bugs which happen in normal web browsers (not Brave). Brave ignores web standards and we gave up fighting with Brave developers a long time ago.

**Comment by abandon-hope** [comment]
This is all specified with as much precision as is possible in my first two messages in this thread. Your responses have given me no confidence that anything I say is being taken at face-value. If you can't be bothered to solve a bug that only affects one user, just say so. If you do want to solve this bug - the info's all here, good luck, I'll be elsewhere.

**Comment by photopea** [comment]
I need to see a bug happen on my computer to be able to fix it.

Could you maybe record a short video of what exactly do you do after opening www.photopea.com?
